.github/codeowners

```diff
@@ -1,3 +1,3 @@
-
 # List of approvers/reviewers for CoT website-v2
-* @ann-codes
\ No newline at end of file
+
+@ann-codes @anacestoque @apple-orangez @masuyama13 @bpb2008 @Mahiambaw
```

Comment:
Ah sorry. It has errors
<img width="630" alt="image" src="https://github.com/user-attachments/assets/2158b03d-43a8-4836-969d-c2d8dc8ba2fc" />

Reply:
Yeah I saw that too, I already sent Ana DM about it. She still needs to accept the invite. But I am assuming it will happen soon so it should be ok